src/screens/admin/FieldMappingConfig.tsx

```
@@ -0,0 +1,1046 @@
+				isClosable: true,
+			});
+		}
+	};
+
+	// --- Helper to parse VC fields from document config ---
+	function parseVcFields(vcFieldsRaw) {
+		if (!vcFieldsRaw) return [];
+		try {
+			const parsed = JSON.parse(vcFieldsRaw);
+			return Object.entries(parsed).map(([key, value]) => {
+				const v = value as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)